examples/ansible/setup_neuvector.yml

```diff
@@ -0,0 +1,109 @@
+---
+# This Ansible playbook is used to manage the NeuVector workload on a ALP host.
+# The playbook supports two operations: setup and removal of NeuVector.
+# The operation is determined by the 'neuvector_state' variable, which can be set to 'present' or 'absent'.
+# When 'neuvector_state' is set to 'present', the playbook will:
+#   - Check if the tools for the NeuVector workload are already installed
+#   - Set SELinux into permissive mode
+#   - Retrieve the NeuVector image
+#   - Install the necessary tools for the NeuVector workload
+#   - Start the NeuVector service
+#   - Enable the NeuVector service to start on a reboot
+#   - Print a message with the connection details for NeuVector
+# When 'neuvector_state' is set to 'absent', the playbook will:
+#   - Stop the NeuVector service
+#   - Uninstall the tools for the NeuVector workload
+#   - Remove the NeuVector image
+
+- name: Running the NeuVector workload
+  hosts: alphost
+  vars:
+    neuvector_state: present
+    workload:
+      name: neuvector
+      image: registry.opensuse.org/suse/alp/workloads/bci_containerfiles/suse/alp/workloads/neuvector-demo:latest
+
+  tasks:
+    - name: Setup NeuVector
+      when:
+        - neuvector_state == "present"
+      block:
+        - name: Check if tools for workload {{ workload.name }} are already installed
+          become: true
+          ansible.builtin.command: podman container inspect {{ workload.name }}-allinone
+          register: workload_installed
+          changed_when: false
+          ignore_errors: true
+
+        - name: Set SELinux into permissive mode
+          become: true
+          ansible.builtin.command: setenforce 0
+          when:
+            - workload_installed is failed
+
+        - name: Retrieve image for workload {{ workload.name }}
+          become: true
+          containers.podman.podman_image:
```

Comment:
And you are using containers.podman.podman_image here, so probably better to implement the image inspection above using the appropriate Ansible action rather than via a command line.